chamber/management/commands/makemessages.py

```diff
@@ -1,6 +1,19 @@
+import os
+
 from django.core.management.commands import makemessages
 
 
+def _remove_pot_creation_date(pofile):
+    if os.path.exists(pofile):
+        with open(pofile, "rb+") as f:
+            lines = f.readlines()
+            f.seek(0)
+            for line in lines:
+                if not line.startswith(b'"POT-Creation-Date: '):
+                    f.write(line)
```

Comment:
remove creation date from po file

Reply:
OMG, i didnt know you must do it so manually..

Reply:
how is this not built in?

Reply:
Wouldn't this end up with the last line being in the file twice? the original and the one written over the second to last? You are writing to the original file from the start.

Reply:
yes gettext has no parameter for it

no for this reason there is truncate. Truncate removes next lines from the cursor

Reply:
oh ok